### PR TITLE
Profile token and graph contracts -- WIP

### PR DESCRIPTION
**Pull request by ewerx**

### CODE REVIEW

File: contracts/ProfileGraph.sol [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+    // data
+
+    //mapping(uint256 => uint256[]) public _followers;
+    //mapping(uint256 => uint256[]) public _following;
+
+    // events
+
+    event NewFollow(uint256 follower, uint256 following);
+    event UnFollow(uint256 follower, uint256 following);
```
Comment (gabsong) [inline contracts/ProfileGraph.sol]: nit: I think we can just call them Follow / Unfollow to simplify. But I see why, we have functions of the same name (lowercase)

Comment (gabsong) [reply contracts/ProfileGraph.sol]: Also maybe instead of follower we could call it target, and instead of following call it requester

**Review by gabsong — APPROVED** [review]
LGTM

File: contracts/ProfileGraph.sol (hunk 1 above)
Comment (ewerx) [reply contracts/ProfileGraph.sol]: made it (follower,  followed) and (follower, unfollowed)

**Pull request merged**